src/clients/python/experimental_api_v2/library/grpcclient.py

```diff
@@ -270,7 +270,7 @@ def get_server_metadata(self, headers=None, as_json=False):
             if self._verbose:
                 print(response)
             if as_json:
-                return json.loads(MessageToJson(response))
+                return json.loads(MessageToJson(response), preserving_proto_field_nam=True)
```

Comment:
`preserving_proto_field_name` should be parameter for `MessageToJson()`?

Reply:
nam -> name
@GuanLuo are you referring to that misspelling?

Reply:
Fixed. That and the fact that `preserving_proto_field_name` is a param to `MessageToJson()` and not `json.loads()`